samples/drivers/watchdog/Makefile

```diff
@@ -1,4 +1,12 @@
 BOARD ?= quark_d2000_crb
-CONF_FILE = prj.conf
+
+ifeq ($(BOARD),96b_carbon)
+CONF_FILE ?= prj_$(BOARD).conf
+else
+ifeq ($(BOARD),nucleo_f334r8)
+CONF_FILE ?= prj_$(BOARD).conf
```

Comment:
@erwango Please take a look at the last modified commit. Let me know if such an approach is acceptable to build the sample application.